### PR TITLE
Minor edits from read through

### PR DESCRIPTION
**Pull request by jussmi**

### CODE REVIEW

File: {{cookiecutter.app_slug}}/tests/test_{{cookiecutter.app_module}}.py [hunk 1]
```diff
@@ -31,4 +31,4 @@ def test_api_echo(client):
 def test_api_echo_empty(client):
     """Should return a 404 error"""
     resp = client.get('/api/echo')
-    assert resp.status_code == 404
\ No newline at end of file
+    assert resp.status_code == 404
```
Comment (jussmi) [inline {{cookiecutter.app_slug}}/tests/test_{{cookiecutter.app_module}}.py:34]: I'm not sure what this one is. I don't remember changing anything here.

Comment (ericdill) [reply {{cookiecutter.app_slug}}/tests/test_{{cookiecutter.app_module}}.py:34]: it's a new line that I'm reasonably certain your editor automatically added when you saved the file

File: {{cookiecutter.app_slug}}/{{cookiecutter.app_module}}/_version.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def get_config():
     cfg.style = "pep440"
     cfg.tag_prefix = ""
     cfg.parentdir_prefix = "None"
-    cfg.versionfile_source = "audience_utils/_version.py"
```
Comment (ericdill) [inline {{cookiecutter.app_slug}}/{{cookiecutter.app_module}}/_version.py:46]: oops looks like that internal top sekret maxpoint code leaked out 😝

Comment (parente) [reply {{cookiecutter.app_slug}}/{{cookiecutter.app_module}}/_version.py:46]: Oh darn. Interesting why that's even in there considering it started from a fresh project.

**Pull request merged**